### PR TITLE
ESM

### LINKED ISSUE #59: Move to ESM
> @GrahamSH-LLK We can't merge this because we would need to import chalk using ESM which isn't what we use :|
> See https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c

I've been meaning to 100% move to ESM. It's so much cleaner and nicer.

_Originally posted by @GrahamSH-LLK in https://github.com/1hostjs/hosting/issues/58#issuecomment-821125874_

### PR DESCRIPTION
**Pull request by wgyt**
**Resolves**

Resolves #59

**Changes**

Move to ESM

**Tests**

Not working yet

### CODE REVIEW

**Comment by wgyt** [comment]
@GrahamSH-LLK I almost got it to work, but I need the modules to use import. Would that be an issue?

**Comment by GrahamSH-LLK** [comment]
> @GrahamSH-LLK I almost got it to work, but I need the modules to use import. Would that be an issue?

No

File: src/formatting.js [hunk 1]
```diff
@@ -35,10 +35,10 @@ function foldLines(
...
 function colorize(text) {
   return text
-    .replace(/\[[^ ]+]/g, (m) => chalk.grey(m))
-    .replace(/<[^ ]+>/g, (m) => chalk.green(m))
-    .replace(/ (-[-\w,]+)/g, (m) => chalk.bold(m))
-    .replace(/`([^`]+)`/g, (_, m) => chalk.bold.cyan(m));
+    .replace(/\[[^ ]+]/g, (m) => grey(m))
+    .replace(/<[^ ]+>/g, (m) => green(m))
+    .replace(/ (-[-\w,]+)/g, (m) => bold(m))
+    .replace(/`([^`]+)`/g, (_, m) => bold.cyan(m));
```
Comment (GrahamSH-LLK) [inline src/formatting.js:41]: How does this work? Chalk is still in chalk.X

Comment (wgyt) [reply src/formatting.js:41]: I don't know, we don't even use colorize

**Comment by wgyt** [comment]
@GrahamSH-LLK this PR almost works, but import.js doesn't return the data properly. Could you help fix this?

**Comment by wgyt** [comment]
Currently this is the error:
```sh
gitpod /workspace/hosting/tests/posix $ 1host

   ╭────────────────────────╮
   │                        │
   │   ✔ 1host.js Success   │
   │                        │
   │   Starting...          │
   │                        │
   ╰────────────────────────╯


Serving
TypeError: moduless is not iterable
    at Server.<anonymous> (/workspace/hosting/src/serve.js:33:26)
    at Server.emit (events.js:315:20)
    at parserOnIncoming (_http_server.js:874:12)
    at HTTPParser.parserOnHeadersComplete (_http_common.js:126:17)
/workspace/hosting/src/serve.js:44
          moduless.errorHandler.module(req, res, 500);
                   ^

TypeError: Cannot read property 'errorHandler' of undefined
    at Server.<anonymous> (/workspace/hosting/src/serve.js:44:20)
    at Server.emit (events.js:315:20)
    at parserOnIncoming (_http_server.js:874:12)
    at HTTPParser.parserOnHeadersComplete (_http_common.js:126:17)
 ```

**Pull request merged**